cmd/lakefs/cmd/run.go

```diff
@@ -254,26 +250,7 @@ var runCmd = &cobra.Command{
 		defer closeOtfService()
 
 		// start API server
-		apiHandler := api.Serve(
-			cfg,
-			c,
-			middlewareAuthenticator,
-			authService,
-			blockStore,
-			authMetadataManager,
-			migrator,
-			bufferedCollector,
-			cloudMetadataProvider,
-			actionsService,
-			auditChecker,
-			logger.WithField("service", "api_gateway"),
-			emailer,
-			templater,
-			cfg.Gateways.S3.DomainNames,
-			cfg.UISnippets(),
-			upload.DefaultPathProvider,
-			otfDiffService,
-		)
+		apiHandler := api.Serve(cfg, c, middlewareAuthenticator, authService, blockStore, authMetadataManager, migrator, bufferedCollector, cloudMetadataProvider, actionsService, auditChecker, logger.WithField("service", "api_gateway"), emailer, cfg.Gateways.S3.DomainNames, cfg.UISnippets(), upload.DefaultPathProvider, otfDiffService)
```

Comment:
Liked the old multi line formatting better.

Reply:
Probably [gofumpt](https://github.com/mvdan/gofumpt) I'll see what I can do